datanode/sqlstore/proposals.go

```diff
@@ -266,7 +267,7 @@ func (ps *Proposals) Get(ctx context.Context,
 	}
 
 	if proposalType != nil {
-		conditions = append(conditions, fmt.Sprintf("terms ? %s", nextBindVar(&args, *proposalType)))
+		conditions = append(conditions, fmt.Sprintf("terms ? %s", nextBindVar(&args, proposalType.String())))
```

Comment:
Maybe theres already one that exists and I've just not see it, but can we have a test in `proposal_test.go` for filtering on proposal test?